src/style.css

```diff
@@ -2,2 +1,76 @@
+html {
+  font-family: Roboto, sans-serif;
+  font-style: normal;
+  line-height: normal;
+  font-size: 12px;
+  font-weight: 500;
+  text-transform: uppercase;
```

Comment:
Applying `text-transform: uppercase;` globally to the `html` element might be too restrictive. Consider applying this style to specific elements that require uppercase text to allow for more flexible text styling across the site.